WORKSPACE

```diff
@@ -195,8 +196,13 @@ go_register_toolchains(nogo = "@com_github_cockroachdb_cockroach//:crdb_nogo")
 # begin rules_nodejs dependencies #
 ###################################
 
+# Install rules_nodejs dependencies
+load("@build_bazel_rules_nodejs//:repositories.bzl", "build_bazel_rules_nodejs_dependencies")
+build_bazel_rules_nodejs_dependencies()
```

Comment:
BEFORE the call to `build_bazel_rules_nodejs_dependencies()`, please add the following comments:

```
# bazel_skylib handled above.
# rules_nodejs handled above.
```

This mirrors other comments we have in `WORKSPACE`. Basically these comments give us (the maintainers) confidence that we have manually checked the dependencies and mirrored all of them.